### PR TITLE
routine

### PR DESCRIPTION
**Pull request by l-goede**

### CODE REVIEW

**Review by tmh9x — APPROVED** [review]

File: src/utils/questions.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  const answers = await inquirer.prompt<{ service: string }>({
+    type: 'list',
+    name: 'service',
+    message: 'Please choose service',
+    choices: services,
+  });
+  return answers.service;
+};
+
+export const addNewCredential = async (): Promise<Credential> => {
```
Comment (christophersiem) [inline src/utils/questions.ts:37]: maybe rename to `askForCredential`

**Review by christophersiem — APPROVED** [review]
nice 🎉

File: src/utils/questions.ts (hunk 1 above)
Comment (l-goede) [reply src/utils/questions.ts:37]: I've changed it , thanks!!

**Review by christophersiem — APPROVED** [review]
awesome!

**Pull request merged**